kernel/CMakeLists.txt

```diff
@@ -52,7 +52,10 @@ target_sources_ifdef(
   userspace.c
   )
 
+get_property(E_KERNEL_ENTRY GLOBAL PROPERTY E_KERNEL_ENTRY)
```

Comment:
You forgot to remove this from the root.

Reply:
will remove.